test/correctness/simplify.cpp

```diff
@@ -305,6 +305,66 @@ void check_algebra() {
     check((7 - y) / 7, (-y) / 7 + 1);
     check((y - 7) / 7, y / 7 + (-1));
 
+    // TODO: The commented cases below should be handled by
```

Comment:
Doesn't seem like there are commented cases anymore, so this TODO might be able to be removed?